ckanext/canada/templates/internal/recombinant/resource_edit.html

```diff
@@ -18,7 +18,7 @@
     <div class="panel panel-primary">
       <div class="panel-heading">{{ _("Guide") }}</div>
       <div class="panel-body">
-        <p>{% trans -%}For information on how to prepare, maintain and upload templates to the Registry, refer to the appropriate documentation posted on <a href="http://www.gcpedia.gc.ca/wiki/Proactive_Disclosure_on_Open.Canada.ca">GCPEDIA</a>{% endtrans %}</p>
+        <p>{% trans -%}For information on how to prepare, maintain and upload templates to the Registry, refer to the appropriate documentation posted on <a href=" http://www.gcpedia.gc.ca/wiki/Open_Data_Inventory_/l%E2%80%99inventaire_dedonn%C3%A9es_ouvertes">GCPEDIA</a>{% endtrans %}</p>
```

Comment:
extra space in the href=""?

This is in a {% trans %} block so the .po file should be updated too